src/Dropdown.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react"
+
+const  dropdownData = ['aaple','banana','orange','litchi']
```

Comment:
make component dynamic and reusable , dont pass the hard coded data , can pass the data through props